doc/rpm-plugin-syslog.8

```diff
@@ -0,0 +1,21 @@
+.TH "RPM-SYSLOG" "8" "14 Apr 2016" "Red Hat, Inc."
+.SH NAME
+rpm-plugin-syslog \- Syslog plugin for the RPM Package Manager
+
+.SH Description
+
+The plugin writes basic information about rpm transactions to the syslog - like transactions run and packages installed or removed.
+
+.SH Configuration
+
+There are currently no options with which the plugin can be configured.
+
+It can be disabled by either setting the \fI%__transaction_syslog\fR macro
+to \fI%{nil}\fR in some \fI/etc/rpm/macros.*\fR file or undefine it
+on the command line with \fB--undefine=__transaction_syslog\fR.
+
+It is also possible to disable all plugins temporarily my passing
+\fB--noplugins\fR to \fBrpm\fR.
+
+Another option is to remove the plugin from the system if it is
+packaged in its own sub package.
```

Comment:
Seems this got forgotten when converting these to "see rpm-plugins(8)"

Reply:
It's done now.